src/config/openApiTypes.ts

```diff
@@ -0,0 +1,51 @@
+interface OpenApiOptions {
```

Comment:
You can import { Options } from "swagger-jsdoc", without creating an interface

Reply:
I deleted interface OpenApiOptions and imported { Options } from "swagger-jsdoc". 
Thank you for sharing this knowledge!